### DISCUSSION TITLE
lf file manager integration Windows

### DISCUSSION THREAD

**Discussion by yashamon** [nushell/nushell#7652] [Q&A]
### Question

How to integrate nu with lf on windows. Setting shell nu works fine, until running any shell command. Which leads to system cannot find the file specified.

**Comment by fdncred** [comment]
What is `lf`? Do you have a link?

If you're setting a shell as your startup shell somehow, you need to make sure you configure your `Path` in the env.nu, otherwise nushell may not know where your programs are.

**Comment by yashamon** [comment]
Lf is a file manager: https://github.com/gokcehan/lf. Basically, it can outsource various shell commands to a chosen shell like pwsh. This is particularly useful for integration with fzf. Now we can set shell to be nu. And I think Lf finds nu without a problem, but then seems unable to pass commands to it. Everything works fine with pwsh or the windows native cmd shell. I think that it is some problem with pwd variables.

**Comment by rgwood** [comment]
This question does not have enough information for us to help you. Consider adding more information and examples.

**Comment by yashamon** [comment]
Hi I don't think this is actually a nu problem. And I don't know how to trace the issue. I was hoping, since lf is a very popular file manager at least on Linux, that someone more experienced knows how to configure more properly.
